server/jetstream_api.go

```diff
@@ -776,6 +776,23 @@ func (s *Server) sendAPIErrResponse(ci *ClientInfo, acc *Account, subject, reply
 	s.sendJetStreamAPIAuditAdvisory(ci, acc, subject, request, response)
 }
 
+const errRespDelay = 500 * time.Millisecond
+
+func (s *Server) sendDelayedAPIErrResponse(ci *ClientInfo, acc *Account, subject, reply, request, response string) {
+	s.startGoRoutine(func() {
+		defer s.grWG.Done()
+		select {
+		case <-s.quitCh:
+		case <-time.After(errRespDelay):
```

Comment:
Only issue here is we do not bail early if asset goes away like stream or consumer.

Reply:
fixed, passing in raft group, will use `rg.node.QuitC()`

Reply:
Asset may not have a raft group, could be R=1.

Reply:
We need the stream and consumer quit chan I think. Or we punt with comment for next release to add in asset tracking for this.

Reply:
wouldn't that channel go away with the stream too?
at this point we do't really have the consumer/stream yet.
I'm not worried about responding for smth. that went away while we delayed. 
I am also convinced that this will see a revamp once we have only one disconnected follower respond.